evergreen/config.yml

```diff
@@ -659,7 +667,7 @@ tasks:
                 export DEVELOPER_DIR="${xcode_developer_dir}"
             fi
 
-            ./evergreen/install_baas.sh -w ./baas-work-dir -b 4da4c3b3e9083ea834522bdf67a2e129fd701d86 2>&1 | tee install_baas_output.log
+            ./evergreen/install_baas.sh -w ./baas-work-dir -b 437b21ea4994bd5134abad87896d0fcc65b4b61c 2>&1 | tee install_baas_output.log
```

Comment:
Can you update to `773fd0e93fb569b0b4304bb4ca9401f33351e0f1`? Another migration fix went in recently

Reply:
also, does https://github.com/realm/realm-core/blob/78175060de3787e6450005aed4cfb7072619a78e/dependencies.list#L5 need to be updated? or do the sync migration tests not run in jenkins?

Reply:
Yes - thanks for the updates - I'll need to update `MDBREALM_TEST_SERVER_TAG` to today's date, but the build for the commit above won't be created until around 8pm this evening.